### PR TITLE
rewrite: analyze in JS & migrate to oha

### PR DESCRIPTION
**Pull request by neo773**
This PR rewrites Analysis code in JS and migrates `wrk` to [oha](https://github.com/hatoo/oha) it's faster, Rust based and supports outputting as JSON so no more manual parsing txts

### CODE REVIEW

File: oha/bench2.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DURATION="30s"
+QUERY='{"operationName":null,"variables":{},"query":"{posts{id,userId,title,user{id,name,email}}}"}'
+
+oha -z $DURATION -c 100 -m POST \
```
Comment (tusharmath) [inline oha/bench2.sh:6]: why are we moving to oha?

File: run_benchmarks.sh [hunk 2]
```diff
@@ -69,5 +69,5 @@ for service in "apollo_server" "caliban" "netflix_dgs" "gqlgen" "tailcall" "asyn
     fi
 done
 
-bash analyze.sh "${bench1Results[@]}"
-bash analyze.sh "${bench2Results[@]}"
+node analyze.js "${bench1Results[@]}"
+node analyze.js "${bench2Results[@]}"
```
Comment (tusharmath) [inline run_benchmarks.sh:73]: run_benchmarks.sh should also move to node.js

**Comment by tusharmath** [comment]
Moving to draft to reduce noise and improve CI efficiency. Once you are ready just mark it as "ready to review". Feel free to give a shoutout on the #contributors channel on Discord if you want immediate attention.

**Comment by neo773** [comment]
Superseded by https://github.com/tailcallhq/graphql-benchmarks/pull/316

**Pull request closed without merging**